app/assets/stylesheets/pages/_crud_employee.css.scss

```diff
@@ -36,6 +36,12 @@
 
```

Comment:
#### Incorrect Identation

#### Some blocks are not ordered alphabetically.

![image](https://user-images.githubusercontent.com/73558208/214730899-f2ac460b-f564-4e19-b348-0f41c93dca01.png)

## 
![image](https://user-images.githubusercontent.com/73558208/214731221-64cf7db6-15eb-4b20-92fc-9ddaa8ddfcba.png)

Reply:
Done